server/controllers/pages.js

```diff
@@ -16,7 +16,27 @@ const entryHelper = require('../helpers/entry')
  * Edit document in Markdown
  */
 router.get('/edit/*', (req, res, next) => {
-  if (!res.locals.rights.write) {
+  var xPaths = ['news', 'general'] // the final word in the url For.e.g : news in /edit/news, general in /edit/general 
+
+  var checkEditAvailable = function (xPaths) {
+    let paths = req.path.split('/')
+
+    let path = paths[ paths.length - 1 ]
+
+    for (let i = 0; i < xPaths.length; i++) {
```

Comment:
How about using `xpaths.forEach()` instead of a regular for loop? They are both skmilar but that sort of males for a better reading experience and is the more *javascripty* way.

Reply:
I tried for forEach initially but that was just returning from forEach function and not the actual function we intend to.

Reply:
Ah, I see. Cool then.